SCClassLibrary/Common/Quarks/Git.sc

```diff
@@ -48,6 +48,22 @@ Git {
 		});
 		^nil
 	}
+	httpsString {
+		var giturl = this.url;
+		var hosturl, path;
+
+		if(giturl.notNil, {
```

Comment:
this is tightly coupled to github, which i understand is the intention, but let's make sure that what we're dealing with here is actually a github URL. maybe `giturl.beginswith("git@github.com:")` and `giturl.beginsWith("git://github.com/")`? if it isn't identifiably a github URL, then we should throw an error.